cupy/cuda/cufft.pyx

```diff
@@ -59,16 +68,60 @@ cpdef inline check_result(int result):
         raise CuFftError(result)
 
 
-cpdef destroy(size_t plan):
-    result = cufftDestroy(plan)
-    check_result(result)
-
-
-cpdef plan1d(int nx, int type, int batch):
-    cdef Handle plan
-    result = cufftPlan1d(&plan, nx, <Type>type, batch)
-    check_result(result)
-    return plan
+class Plan1d(object):
```

Comment:
`scikit.cuda` provides `Plan` calss. [link](http://scikit-cuda.readthedocs.io/en/latest/generated/skcuda.fft.Plan.html)
Which do you think is better?

Reply:
I think Plan1d is the better choice in the current implementation.
cuFFT supports 1D/2D/3D/Many plans, therefore we should clarify which one we are using.
If we support other plans or the general plan in the future, we will add or rename classes.

Reply:
OK, I understand.